src/awkward/_connect/cuda/cuda_kernels/awkward_ListArray_getitem_next_array_advanced.cu

```diff
@@ -0,0 +1,45 @@
+// BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+
+enum class LISTARRAY_GETITEM_NEXT_ARRAY_ADVANCED_ERRORS {
+  STOP_LT_START,     // message: "stops[i] < starts[i]"
+  STOP_GET_LEN,      // message: "stops[i] > len(content)"
+  IND_OUT_OF_RANGE,  // message: "index out of range"
+};
```

Comment:
Is there Python code that parses the comment to generate the error message in Python? Does it also check the correct `enum class`, so that it gets error messages associated with this function, and not, for example, `LISTARRAY_GETITEM_NEXT_AT_ERRORS`?

Reply:
I don't see any code that does that. It was done similarly in other kernels. But I think whoever wrote it might have intended to that.

Reply:
Also, I did find some functions that try to parse these messages but they don't seem to be working properly. I need to check further.